src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -580,9 +579,10 @@ public KubernetesClient connect() throws KubernetesAuthException, IOException {
      * Check not too many already running.
      *
      */
-    private boolean addProvisionedSlave(@Nonnull PodTemplate template, @CheckForNull Label label, int scheduledCount) throws Exception {
+    private boolean addProvisionedSlave(@Nonnull PodTemplate template, @CheckForNull Label label, int numProvisioned,
+                                        int numInProvisioning) throws Exception {
         if (containerCap == 0) {
-            return true;
+            return false;
```

Comment:
Should be in another PR

Reply:
Please move this one to another PR for clarity.

Reply:
OK, will do.

Reply:
Moved to https://github.com/jenkinsci/kubernetes-plugin/pull/828